tools/image-info

```diff
@@ -533,7 +518,8 @@ def read_services(tree, state):
         "chrony-wait.service"
     ]
     """
-    services_state = subprocess_check_output(["systemctl", f"--root={tree}", "list-unit-files"], (lambda s: parse_unit_files(s, state)))
+    services_state = subprocess_check_output(
+        ["systemctl", f"--root={tree}", "list-unit-files"], (lambda s: parse_unit_files(s, state)))
```

Comment:
This commit mixes functional and codestyle changes.  I'd like them separate for a cleaner history.
It was generally quite hard to read this patch and I'm not entirely sure I followed most of it.

Reply:
Indeed this commit is badly split up. I'm gonna fix that.

Reply:
I'm gonna split this in several ones, they won't work without each other but it's gonna be easier to read through it.